sdk/core/azure-core/src/test/java/com/azure/core/util/polling/Response.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.util.polling;
+
+/**
+ * A test response type for testing polling.
+ */
+public class Response {
```

Comment:
Let's call this `TestResponse` so it doesn't show up on IntelliJ when searching for `Response` class in src/main.